src/components/pages/roster-page/roster-page.tsx

```diff
@@ -104,18 +110,21 @@ export const RosterPage = () => {
 
   const reloadTable = useCallback(async () => {
     try {
-
+      let sortParams = '';
+      if (sortState) {
+        sortParams = `&orderBy=${sortState.column}&sortDirection=${sortState.sortDirection}`;
```

Comment:
I've started using the `params` option in axios requests, which I think makes for cleaner and less error prone code. There's an example of how to use it in `muster-page.tsx` line 146.